src/org/spdx/rdfparser/license/SimpleLicensingInfo.java

```diff
@@ -44,18 +46,20 @@
 	protected String comment;
 	protected String name;
 	protected String[] seeAlso;
-	
+	// protected String[] seeAlsoDetails;
+	protected JSONArray seeAlsoDetails;
```

Comment:
Minor comment - looks like the comment above could be removed

Reply:
Thanks for catching that